src/Definition/Common.php

```diff
@@ -58,6 +72,22 @@ public function isNullable()
     /**
      * @return string
      */
+    public function getDefault()
```

Comment:
It uses the `getSQLDefinition`, but if I don't feed them the format/default, the result will be the same. The format/default may also incur validations (in the ctor), that will be backend specific.